### ISSUE TITLE
Typelib file for namespace 'GnomeKeyring' (any version) not found

### ISSUE THREAD

**Issue by cyclotron3k** [mvz/gir_ffi-gnome_keyring#6]
On Ubuntu 20.04 / Ruby 2.7.0, having some issues using `gir_ffi-gnome_keyring`.

`require 'gir_ffi-gnome_keyring'` fails, and gives:

```
RuntimeError: Typelib file for namespace 'GnomeKeyring' (any version) not found
from /var/lib/gems/2.7.0/gems/gir_ffi-0.15.3/lib/ffi-gobject_introspection/i_repository.rb:42:in `require'
Caused by LoadError: cannot load such file -- gir_ffi-gnome_keyring
from /usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:92:in `require'
```

I'm still digging into it. `gnome-keyring-daemon` appears to be running fine.

**Comment by mvz** [comment]
You'll have to install the gir1.2-gnomekeyring-1.0 package.

**Comment by mvz** [comment]
Note that this package is only available in Ubuntu up to 18.04.

**Comment by cyclotron3k** [comment]
Ok, thanks @mvz, that's good to know.
